win32/zlib-ng.def.in

```diff
@@ -45,11 +50,6 @@ EXPORTS
     @ZLIB_SYMBOL_PREFIX@zng_adler32_combine
     @ZLIB_SYMBOL_PREFIX@zng_crc32_combine
 ; various hacks, don't look :)
```

Comment:
Is this comment still needed?

Reply:
It still relates to the rest of the below functions, same as in zlib(-ng).h